# Generated by Django 3.1.1 on 2020-11-04 15:18

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('preguntas', '0007_auto_20201104_0906'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='pregunta',
            name='puntosNegativos',
        ),
        migrations.AddField(
            model_name='pregunta',
            name='puntosNegativos',
            field=models.ManyToManyField(blank=True, related_name='preguntasPuntosNegativos', to=settings.AUTH_USER_MODEL),
        ),
        migrations.RemoveField(
            model_name='pregunta',
            name='puntosPositivos',
        ),
        migrations.AddField(
            model_name='pregunta',
            name='puntosPositivos',
            field=models.ManyToManyField(blank=True, related_name='preguntasPuntosPositivos', to=settings.AUTH_USER_MODEL),
        ),
        migrations.RemoveField(
            model_name='respuesta',
            name='puntosNegativos',
        ),
        migrations.AddField(
            model_name='respuesta',
            name='puntosNegativos',
            field=models.ManyToManyField(blank=True, related_name='resouestasPuntosNegativos', to=settings.AUTH_USER_MODEL),
        ),
        migrations.RemoveField(
            model_name='respuesta',
            name='puntosPositivos',
        ),
        migrations.AddField(
            model_name='respuesta',
            name='puntosPositivos',
            field=models.ManyToManyField(blank=True, related_name='respuestasPuntosPositivos', to=settings.AUTH_USER_MODEL),
        ),
    ]
